src/more_click/options.py

```diff
@@ -55,3 +58,14 @@ def _number_of_workers() -> int:
 )
 force_option = click.option('-f', '--force', is_flag=True)
 debug_option = click.option('--debug', is_flag=True)
+
+# sorted level names, by log-level
+_level_names = [k for k, _ in sorted(logging._nameToLevel.items(), key=itemgetter(1))]
```

Comment:
this is evil and I love it. however, there's a better way

```suggestion
 _level_names = sorted(logging._nameToLevel, key=logging._nameToLevel.get)
```